scripts/analysis.py

```diff
@@ -356,10 +351,190 @@ def plot_in_out_flux(cursor, facility, influx_bool, title, outputname):
                           title, outputname, init_year)
     else:
         multi_line_plot(waste_dict, timestep,
-                       'Years', 'Mass [kg]',
+                        'Years', 'Mass [kg]',
                         title, outputname, init_year)
 
 
+def total_waste_timeseries(cursor):
+    """Plots a stacked bar chart of the total waste mass vs time
+
+    Parameters
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    null
+    stacked bar chart of waste mass vs time
+    """
+
+    cur = cursor
+    agent_ids = get_agent_ids(cur, 'sink')
+    resources = cur.execute(exec_string(agent_ids,
+                                        'transactions.receiverId',
+                                        'sum(quantity), senderid, time')
+                            + ' GROUP BY time, senderid').fetchall()
+    init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+    waste_dict = collections.OrderedDict({})
+
+    spec_list = []
+    from_reactor = 0
+    from_fuelfab = 0
+    from_separations = 0
+    from_enrichment = 0
+
+    reactor_timeseries = []
+    separations_timeseries = []
+    enrichment_timeseries = []
+
+    for i in range(0, duration):
+        for row in resources:
+            transaction_time = row[2]
+            if transaction_time == i:
+                senderid = row[1]
+                quantity = row[0]
+                spec = cur.execute("""SELECT spec from
+                                    agententry WHERE
+                                    agentid =""" + str(row[1])).fetchone()
+                if "Reactor" in spec[0]:
+                    from_reactor += quantity
+                elif "Enrichment" in spec[0]:
+                    from_enrichment += quantity
+                elif "Separations" in spec[0]:
+                    from_separations += quantity
+        reactor_timeseries.append(from_reactor/1000)
```

Comment:
no, it just looks a little arbitrary in an analysis script to be dividing my 1000 at random. Three common ways to do this and maintain clarity of intent (rather than raise suspicion) are:

1) Good way: use meaningful variable names:

```python
mthm_from_reactor = kg_from_reactor/1000.0
```
2) Better way: Create a function that does this and use it:

```python
def kg_to_mthm(kg_amt):
    return kg_amt/1000.0
...
kg_to_mthm(from_reactor)

```

3) Use a unit handling library like pint https://pint.readthedocs.io/en/0.7.2/


Finally, note how I use 1000.0 rather than the integer 1000. This is for python2.7 compatibility. Without the decimal, you may lose information for some divides (because python2.7 will return an integer if you divide one integer by another. In python2.7 1/2=0, 4/3=1, and 20/1000 = 0)